bitvm/src/chunk/g16_runner_core.rs

```diff
@@ -326,10 +326,10 @@ pub(crate) fn groth16_generate_segments(
     push_compare_or_return!(t4);
 
     // (t2, t3) = (le.t2, le.t3);
-    let tmp_q2f = frob_q_power(pubs.q2, -1);
-    t2 = (t2 + tmp_q2f).into_affine();
-    let tmp_q3f = frob_q_power(pubs.q3, -1);
-    t3 = (t3 + tmp_q3f).into_affine();
+    // let tmp_q2f = frob_q_power(pubs.q2, -1);
+    // t2 = (t2 + tmp_q2f).into_affine();
+    // let tmp_q3f = frob_q_power(pubs.q3, -1);
+    // t3 = (t3 + tmp_q3f).into_affine();
```

Comment:
can delete these comments too, but i'm okay either way.

Reply:
Thanks @manishbista28 for checking this. I recommend deleting instead of commenting. We have them in the repository history if needed.

Reply:
Same with lines 1058-1062 in the same file?

Reply:
yes, same for comments in lines 1058-1062.